### DISCUSSION TITLE
Zebra Programming Language (ZPL) Code Identificaiton.

### DISCUSSION THREAD

**Discussion by Bamuel** [community/community#106557] [Repositories]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/6744113/9767a15b-1c72-43b6-b18b-028d0b89a846)
I have create a new repo using Zebra Programming Language ZPL.

However github has incorrectly identified the wrong programming code language.
https://github.com/Bamuel/ZPL-Label-Designs

**Comment by spenserblack** [comment]
GitHub uses [linguist](https://github.com/github-linguist/linguist) to detect the programming language. If you want to add a new language, you can open up a PR. However, linguist requires the language to be reasonably popular before they add support (check their `CONTRIBUTING.md` for more details). Here's a quick search for the language's popularity I did, but you might be able to find more files by modifying this: https://github.com/search?q=path%3A*.zpl+%2F%5E%5C%5E%2F+NOT+is%3Afork&type=code

Is there another language that's similar to Zebra? You might want to override the detected language if so. Note that you can only override with languages that are supported by linguist.

```gitattributes
*.zpl linguist-language=Name-of-Similar-Language
```

**Reply by Bamuel** [reply]
Thanks. Will look into linguist and do a PR soon.
